src/lib/leaflet.js

```diff
@@ -13,4 +13,5 @@ export const rentStrikeIcon = new L.Icon({
   className: "icon-rent-strike",
 });
 
+window.L = L;
```

Comment:
Why was this needed?

Reply:
Ah that’s left over from when I was trying to debug the Leaflet Pattern plug-in. I’ll remove it before merging. Thanks for catching.